topi/tests/python/test_topi_vision.py

```diff
@@ -43,7 +43,7 @@ def check_device(device):
         f(tvm_data, tvm_valid_count, tvm_out)
         tvm.testing.assert_allclose(tvm_out.asnumpy(), np_result, rtol=1e-4)
 
-    for device in ['llvm', 'opencl']:
+    for device in ['llvm', 'opencl', 'cuda']:
```

Comment:
@Laurawly Does nms working on cpu now? I think we still need graph annotation to run ssd on gpu, due to issue in cpu sorting implementation?